### PR TITLE
[ICP-9251]change smoke sensor health check period to 12mins

### PR DESCRIPTION
**Pull request by FenMei007S**
The smoke sensor health check period 6 hours too long, HQ request change to 22mins, @tpmanley  plz review it.

### CODE REVIEW

File: devicetypes/smartthings/zigbee-smoke-sensor.src/zigbee-smoke-sensor.groovy [hunk 1]
```diff
@@ -135,11 +135,11 @@ def ping() {
 }
 def configure() {
 	log.debug "configure"
-	sendEvent(name: "checkInterval", value:6 * 60 * 60 + 2 * 60, displayed: false, data: [protocol: "zigbee", hubHardwareId: device.hub.hardwareID, offlinePingable: "1"])
+	sendEvent(name: "checkInterval", value:20 * 60 + 2 * 60, displayed: false, data: [protocol: "zigbee", hubHardwareId: device.hub.hardwareID, offlinePingable: "1"])
```
Comment (tpmanley) [inline devicetypes/smartthings/zigbee-smoke-sensor.src/zigbee-smoke-sensor.groovy]: ZoneStatus is configured to report every 180 seconds so it shouldn't take 20 minutes to determine the device is offline. I suggest setting `checkInterval` to 12 minutes like most other Zigbee devices, even though this one is configured to report a little more often (3 minutes for this device vs 5 minutes for most other Zigbee devices).

**Comment by tpmanley** [comment]
Also, if you're going to need this in staging before next Tuesday please point the PR to the `staging` branch.

**Comment by FenMei007S** [comment]
@tpmanley  the proid change to 12mins, plz review

**Comment by tpmanley** [comment]
Changes look good. Can you add a ticket to the PR title?

**Comment by FenMei007S** [comment]
No ticket be registered for this PR

**Comment by tpmanley** [comment]
Can you create one? We prefer to have a ticket for each change

**Comment by FenMei007S** [comment]
Add it

**Review by tpmanley — APPROVED** [review]

**Pull request merged**